birb_registry/src/lib.rs

```diff
@@ -0,0 +1,118 @@
+#![feature(const_trait_impl)]
+#![feature(fs_try_exists)]
+
+use birb::MainThreadModule;
+use std::{collections::HashMap, io::prelude::*, fs::File, fs};
+use std::fmt::{Debug, Formatter};
+use serde::{Serialize, Deserialize};
+
+#[derive(Default)]
+pub struct Registry {
+    ticks: u16,
+    data: HashMap<String, Vec<u8>>
+}
+
+// Implement Registry Methods
+impl Registry {
+    pub fn new() -> Self {
+        Registry::default()
```

Comment:
You can use Self::default() here instead of Registry::default(), clippy will complain about this